src/js/rtc_session.js

```diff
@@ -484,6 +484,38 @@ export default class RtcSession {
     get mediaStream() {
         return this._userAudioStream;
     }
+    pauseLocalVideo() {
+        if (this._streamToBeClosed) {
+            var videoTrack = this._streamToBeClosed.getTracks()[1];
```

Comment:
since you are here, could you add document and parameter checking when client provides stream